app/controllers/miq_ae_customization_controller/custom_buttons.rb

```diff
@@ -114,7 +114,7 @@ def ab_get_node_info(node)
               when "Host"
                 _("Host / Node")
               when "EmsCluster"
-                ui_lookup(:ems_cluster_types => "cluster")
+                _("Cluster / Deployment Role")
```

Comment:
Shouldn't this call `title_for_cluster`?  (I guess same goes for the Host one 2 lines above as well.)